src/components/MultiSelect/MultiSelectItemsCountButton.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { forwardRef } from "react";
+import { useMultiStyleConfig } from "@chakra-ui/react";
+import Button from "../Button/Button";
+import Icon from "../Icons/Icon";
+import { SelectedItems } from "./MultiSelect";
+
+export interface MultiSelectItemsCountButtonProps {
+  id: string;
+  /** The id of the MultiSelect using this button. */
+  multiSelectId: string;
+  /** The label text rendered within the MultiSelect using this button. */
+  multiSelectLabelText: string;
+  /** The open status of the MultiSelect menu. */
+  isOpen: boolean;
+  /** The selected items state (items that were checked by user). */
+  selectedItems: SelectedItems;
```

Comment:
Change this to `selectedItemsCount: string;`
and add `selectedItemsCount: number;` as a prop